nilearn/_utils/cache_mixin.py

```diff
@@ -108,11 +108,11 @@ def cache(func, memory, func_memory_level=None, memory_level=None,
     memory: instance of joblib.Memory or string
         Used to cache the function call.
 
-    func_memory_level: int, optional
+    func_memory_level: int
```

Comment:
Fair enough, but then func_memory_level is optional and the docstring should be consistent.